README.md

```diff
@@ -9,17 +9,63 @@ OmniAuth Strategy for eBay Apps (for using with eBay REST APIs)
 
 ## Installation
 
-TBD
+Add to your Gemfile:
 
+`gem 'omniauth-ebay-oauth'`
+
+Then `bundle install`.
 
 ## Usage
 
-TBD
+```ruby
+use OmniAuth::Builder do
+  provider :ebay, CLIENT_ID, CLIENT_SECRET, callback_url: RU_NAME
+end
+```
+
+Required options:
+  - __CLIENT_ID__, __CLIENT_SECRET__ - Your application's [OAuth credentials](<https://developer.ebay.com/api-docs/static/oauth-credentials.html>) for the environment you're targeting.
+  - __callback_url__  - Your application's [RuName](<https://developer.ebay.com/api-docs/static/oauth-runame.html>) for the environment you're targeting.
+
+Additional options:
+  - __sandbox__ - Are you running your application in [sandbox mode](<https://developer.ebay.com/api-docs/static/sandbox-landing.html>), default __`true`__.
+  - __scope__ - A list of [OAuth scopes](<https://developer.ebay.com/api-docs/static/oauth-details.html#scopes>) that provide access to the interfaces you call.
```

Comment:
Maybe it worth to note that by default that list of scopes is empty and users should provide it if they will use eBay APIs. Or perhaps it will be good to include public access scope `https://api.ebay.com/oauth/api_scope` by default.

Reply:
Not sure about default value, it's omniauth strategy and eBay returns enough information for simple SSO. Add ability to pass scope as array of scopes or string.